lib/is-element-in-view.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
```

Comment:
One of the only problems with not using someone else's code for this - you have to write tests for it!